yb-voyager/src/metadb/migrationStatus.go

```diff
@@ -53,6 +53,8 @@ type MigrationStatusRecord struct {
 	ExportDataSourceDebeziumStarted bool `json:"ExportDataSourceDebeziumStarted"`
 	ExportDataTargetDebeziumStarted bool `json:"ExportDataTargetDebeziumStarted"`
 
+	OnPrimaryKeyConflictAction string `json:"OnPrimaryKeyConflictAction"` // only used in import data or import data file commands
```

Comment:
Should we have separate flags for `import-data-to-target` and `import-data-file` ? Could they conflict? I.e. is it possible to run import-data-file and import-data using the same export-dir? I think it's technically possible...

If we're going with separate MSR records, I'd recommend using this as an opportunity to use a separate struct/json, not MigrationStatusRecord. Perhaps `ImportDataToTargetState` / `ImportDataFileState` ?

Reply:
>  is it possible to run import-data-file and import-data using the same export-dir

Yeah it might be, but do we recommend doing that.

We can take this opportunity to restructure.
But to be same page, is this a case that mandates the restructuring?

Reply:
> if it's not broken (i.e. if technically you can run both import-data-file and import-data using the same export-dir)

I don't know this. Guideline from us is to use separate export-dirs for each migration/bulk-load.
That kind of workflow is not something we recommend.

If we want to allow that, then there are other considerations also - like they should not be allowed parallel (lock file implementation needs enhancement then)